specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -909,7 +928,19 @@
         "Folder",
         "File"
       ],
-      "type": "string"
+      "type": "string",
+      "x-ms-enum": {
+        "name": "StorageInputType",
+        "modelAsString": true,
+        "values": [
```

Comment:
@iscai-msft curious, is the values array necessary? or can it be omitted so no need to maintain `enum` and `s-ms-enum`